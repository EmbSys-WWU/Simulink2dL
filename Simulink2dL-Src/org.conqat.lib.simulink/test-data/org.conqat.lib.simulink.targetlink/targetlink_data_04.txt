struct('description','','stepfunctionname','$S_$B','stepfunctionclass','default','forcerestartfunction',0,'restartfunctionname','RESTART_$S_$B','restartfunctionclass','default','forceinitfunction',0,'initfunctionname','INIT_$S_$B','initfunctionclass','default','forcetermfunction',0,'termfunctionname','TERM_$S_$B','termfunctionclass','default','subsystemid','default','forcereuse',0,'linkfile',0,'functionarglist',{{}},'additionalarguments',{{}},'unscaledarguments',0,'scalingpropagation',{{}},'incrcodegen',0,'ownedmodules',{{}},'destination','','addtestframe',0,'simmode',1,'codecomment','','codefile','','codecoveragelevel',0,'date',730486)