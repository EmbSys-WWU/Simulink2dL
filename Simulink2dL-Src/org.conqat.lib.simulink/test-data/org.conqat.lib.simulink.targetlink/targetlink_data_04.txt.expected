struct (
  'additionalarguments' -> '{{}}'
  'addtestframe' -> '0'
  'codecomment' -> ''
  'codecoveragelevel' -> '0'
  'codefile' -> ''
  'date' -> '730486'
  'description' -> ''
  'destination' -> ''
  'forceinitfunction' -> '0'
  'forcerestartfunction' -> '0'
  'forcereuse' -> '0'
  'forcetermfunction' -> '0'
  'functionarglist' -> '{{}}'
  'incrcodegen' -> '0'
  'initfunctionclass' -> 'default'
  'initfunctionname' -> 'INIT_$S_$B'
  'linkfile' -> '0'
  'ownedmodules' -> '{{}}'
  'restartfunctionclass' -> 'default'
  'restartfunctionname' -> 'RESTART_$S_$B'
  'scalingpropagation' -> '{{}}'
  'simmode' -> '1'
  'stepfunctionclass' -> 'default'
  'stepfunctionname' -> '$S_$B'
  'subsystemid' -> 'default'
  'termfunctionclass' -> 'default'
  'termfunctionname' -> 'TERM_$S_$B'
  'unscaledarguments' -> '0'
)